app/src/main/java/com/github/se/cyrcle/model/address/Address.kt

```diff
@@ -98,4 +101,21 @@ data class Address(
     }
     return shortenString(string.substring(0, lastComma), length)
   }
+
+  /**
+   * Function that format the Address to display its suggestion name.
+   *
+   * @param maxLength the maximum length of the string
+   * @param mode the mode of the suggestion (MapScreen for the map, ListScreen for the list)
+   * @return the suggestion formatted display name
+   */
+  fun suggestionFormatDisplayName(maxLength: Int, mode: String): String {
```

Comment:
Couldn't the mode have been an enum? I mean string works just fine but it feels like a more elegant solution for this sort of thing

Reply:
I added the enum and indeed it makes the code cleaner!